x-pack/plugin/write-load-forecaster/src/main/java/org/elasticsearch/xpack/writeloadforecaster/WriteLoadForecasterPlugin.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.writeloadforecaster;
+
+import org.elasticsearch.cluster.routing.allocation.WriteLoadForecaster;
+import org.elasticsearch.common.settings.ClusterSettings;
+import org.elasticsearch.common.settings.Setting;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.license.License;
+import org.elasticsearch.license.LicensedFeature;
+import org.elasticsearch.plugins.ClusterPlugin;
+import org.elasticsearch.plugins.Plugin;
+import org.elasticsearch.xpack.core.XPackPlugin;
+
+import java.util.Collection;
+import java.util.List;
+
+import static org.elasticsearch.xpack.writeloadforecaster.LicensedWriteLoadForecaster.MAX_INDEX_AGE_SETTING;
+import static org.elasticsearch.xpack.writeloadforecaster.LicensedWriteLoadForecaster.MIN_UPTIME_SETTING;
+
+public class WriteLoadForecasterPlugin extends Plugin implements ClusterPlugin {
+    public static final LicensedFeature.Momentary WRITE_LOAD_FORECAST_FEATURE = LicensedFeature.momentary(
+        null,
+        "write-load-forecast",
+        License.OperationMode.ENTERPRISE
+    );
+
+    public static final Setting<Double> DEFAULT_WRITE_LOAD_FORECAST_SETTING = Setting.doubleSetting(
+        "index.default_write_load_forecast",
```

Comment:
Is this not the override more than the default write load? It seems to override our automated value.